test.sh

```diff
@@ -1,2 +1,2 @@
-flake8 --max-line-length=120 restructuredtext_lint
+flake8 --max-line-length=130 restructuredtext_lint
```

Comment:
What's the reason for the 130 characters? If it's URL's, let's move that line to a `# noqa`